### ISSUE TITLE
Docs for disabling Dark Mode are incomplete

### ISSUE THREAD

**Issue by woodseowl** [livewire/flux#923]
### Flux version

v1.1.2

### Livewire version

v3.5.12

### What is the problem?

The [docs for disabling dark mode](https://fluxui.dev/docs/dark-mode#disabling-dark-mode) explain you should update tailwind.config.js to use `darkMode: null`, however, a `class="dark"` is still added to the root `<html>` element.

### Code snippets

I see this code added by Flux in the generated HTML for a page containing `@fluxStyles`:
```html
<script>
    let appearance = window.localStorage.getItem('flux.appearance') || 'system'

    if (appearance === 'system') {
        appearance = window.matchMedia('(prefers-color-scheme: dark)').matches ? 'dark' : 'light'
    }

    appearance === 'dark' ? document.documentElement.classList.add('dark') : document.documentElement.classList.remove('dark')
</script>
```

### How do you expect it to work?

I'd like to have the documentation also describe how to prevent the class from being added.

The way that I came up to do it is add the following before `@fluxStyles`:
```html
    <script>
        window.localStorage.setItem('flux.appearance', 'none');
    </script>
```

Ideally, this would not be needed because the code would be able to detect that `darkMode: null` is set and then not inject the script. For the mean time, just getting better documentation would help.

### Please confirm (incomplete submissions will not be addressed)

- [x] I have provided easy and step-by-step instructions to reproduce the bug.
- [x] I have provided code samples as text and NOT images.
- [x] I understand my bug report will be closed if I haven't met the criteria above.

**Comment by calebporzio** [comment]
Ahhhh you're right - if someone enables dark mode, then tries to disable it, the key is left in localStorage and that bit of JS will bring ".dark" back when it shouldn't be. hmmm. @joshhanley ?

**Comment by woodseowl** [comment]
> if someone enables dark mode, then tries to disable it, the key is left in localStorage and that bit of JS will bring ".dark" back when it shouldn't be

Well, not quite that. I'm working with an app that specifically should never have dark mode enabled and needs to not have a "dark" class applied universally because we already have that class name in use for other purposes. The code injected by Flux  adds "dark" as a class if someone has their system settings set to dark mode, regardless of whether darkMode is intended to be on within the app overall.

_(As a side note to this, if there are namespace conflicts with "dark" as a class name in the site design, there's not really a way to use Flux dark mode selection. I did look at how Tailwind handles dark mode naming and saw that the dark mode selector could be [customized with a different classname](https://tailwindcss.com/docs/dark-mode#customizing-the-selector). I realize that's a lot of work to apply in Flux and not a solution for what I've reported in this issue, but maybe it's a long term idea.)_

**Comment by igmltd** [comment]
@calebporzio, I do the following, could this be a solution?

https://github.com/livewire/flux/discussions/913

**Comment by joshhanley** [comment]
@calebporzio yeah I think we should update the dark mode detector to account for that, and get it to clear local storage and remove any dark classes if it detects that Tailwind dark mode has been disabled.

**Comment by dicktornfeldt** [comment]
Having similar issue here. My system is set to dark mode, but Im building an app where I dont want the users to be able to switch between dark/light mode. 

Setting tailwind.config.js to `darkMode: null` still adds a `dark` class to the html element which leads to my variant="primary" buttons to look like this instead of having a black background (among other unwanted stuff, nothing is saved in localstorage)
<img width="154" alt="Image" src="https://github.com/user-attachments/assets/d8b4321b-374c-45ac-abab-b0e083aad01f" />

**Comment by woodseowl** [comment]
> [@calebporzio](https://github.com/calebporzio), I do the following, could this be a solution?
> 
> [#913](https://github.com/livewire/flux/discussions/913)

FYI, I had used the darkMode selector setting discussed in #913 as a solution. That used to work, but the most recent Flux (at the time I reported this issue) says to set `darkMode: null` in order to prevent Flux from managing it, but it still injects the script that adds `class="dark"` to the root html element. The setting of the class is where the issue I've reported arises.

**Comment by joshhanley** [comment]
Yeah the dark class should not be applied if the `tailwind.config.js` is set to `darkMode: null`. I've submitted a PR with a fix which will remove the class and also remove it from local storage if `darkMode` is not set to `selector`.

**Comment by alantrotter** [comment]
I think this might be another example of what @dicktornfeldt is mentioning above, styling not working correctly when darkMode is turned off:

with darkMode set to null, the 'current' flux:navlist.item ends up with white text and a white icon on a white background. Inspecting the page and removing 'dark' from the html element fixes the issue and the item displays correctly.

This is what a current home link looks like:

<img width="253" alt="Image" src="https://github.com/user-attachments/assets/949d985d-189c-4f3f-8e03-121c117e13da" />

**Comment by joshhanley** [comment]
PR has been merged, and the fix will be in the next release!
